### DISCUSSION TITLE
Host can not access wg-easy peers, but the wg-easy container on host can

### DISCUSSION THREAD

**Discussion by dexin-peng** [wg-easy/wg-easy#508] [Q&A] [answered]
Say,
wg-easy host 192.168.1.1/32
wg-easy container on the host 10.0.0.111/32
Peers are 10.0.0.0/24

wg-easy host can not ping peers. But inside the container shell, can ping peers.

Is there any way to access peers on wg-easy host?

**Comment by Pleonazm** [comment]
The easiest is to run a container in Network Host mode. The wg0 interface is shared with the host
The host has no access to wg0 interface, and route to 10.0.0.0/24 (only container).
It depends why do you want it.
Can clients reach other clients/server on 10.0.0.0/24 subnet?

**Reply by dexin-peng** [reply]
Yes, clients can reach other clients/server on 10.0.0.0/24 subnet. Everything is good and expected.

But run a container in Network Host mode is difficult and will produce many bugs.
I want a simple command which will give the host 10.0.0.111/32 too, instead of only the wg-easy container 10.0.0.111/32.

I haven't figured out a way to fix this.

**Reply by ghost** [reply, marked as answer]
I solved it by simply create a wireguard-interface on the host to connect to the container via local IP address. Works like a charm.

**Reply by rambonette** [reply]
@Dave-AC1 do you mind showing the steps? was it something done via docker-compose?

**Reply by deulizealand** [reply]
@Dave-AC1 share the step. so we who need it can follow. thanks

**Reply by DMax-YT** [reply]
@rambonette @deulizealand I had the same problem so here's my solutions (you need the last one)
https://superuser.com/questions/1824728/how-to-proxy-requests-into-wg-easy-docker-container/1824944#1824944

**Reply by deulizealand** [reply]
thanks for your reply, i'll try this method asap.

**Reply by discapes** [reply]
Why did I not think of that :smile: such a simple solution.

**Reply by DMax-YT** [reply]
@discapes well, I didn't make that solution, it was [referenced here](https://github.com/wg-easy/wg-easy/discussions/508#discussioncomment-5382125) before. I just gave an instruction how to implement it.

**Reply by deulizealand** [reply]
@DMax-YT  already try it, but still doesnt work.

i try to ping another peer that connect using config in wg-easy from host and virce versa. but no response.

my host : 10.8.0.6
device try to ping : 10.8.0.3

allowed ips (host) : 10.8.0.0/24 

should i change the network of wg easy to host ?

**Reply by DMax-YT** [reply]
@deulizealand it could be because one of your devices doesn't keep connection with wg alive. Try to search about it in google like "wireguard keepalive".

**Reply by deulizealand** [reply]
@DMax-YT it's already there but still, i can't do the ping

**Reply by deulizealand** [reply]
@DMax-YT 
![image](https://github.com/wg-easy/wg-easy/assets/15981200/a5b8e2f0-4e29-4be9-93fa-c98cbdcae775)

all already connected. i can ping from X14 to Sweet and virce versa. but i can't ping to server from X14 or sweet
